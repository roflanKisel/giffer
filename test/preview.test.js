import React from 'react';
import { shallow } from 'enzyme';
import Preview from '../components/preview';

describe('Preview', () => {
  it('should renders correctly errors', () => {
    const wrapper = shallow(<Preview src="source" height="120" width="240" alt="alt" selected />);

    expect(wrapper).toMatchSnapshot();
  });

  it('should receive all props', () => {
    const wrapper = mount(<Preview src="source" height="120" width="240" alt="alt" selected />);

    expect(wrapper.prop('src')).toEqual('source');
    expect(wrapper.prop('height')).toEqual('120');
    expect(wrapper.prop('width')).toEqual('240');
    expect(wrapper.prop('alt')).toEqual('alt');
    expect(wrapper.prop('selected')).toBeTruthy();
  });
});