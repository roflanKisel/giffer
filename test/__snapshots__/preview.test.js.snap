// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preview should renders correctly errors 1`] = `
<preview__StyledImage
  alt="alt"
  height="120"
  selected={true}
  src="source"
  width="240"
/>
`;
